presto-docs/src/main/sphinx/release/release-0.268.rst

```diff
@@ -0,0 +1,31 @@
+=============
+Release 0.268
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add support to use consistent hashing as a hash strategy when affinity scheduling is used. This feature can be turned on by setting ``node-scheduler.node-selection-hash-strategy`` to ``CONSISTENT``. A separate configuration ``node-scheduler.consistent-hashing-min-virtual-node-count`` specifies the minimal number of virtual nodes in the consistent hashing ring.
+* Support Hive scalar function.
```

Comment:
Should this go in Hive changes?

Reply:
Looking at the PR, it's more of a presto change to support hive scalar function. So probably fine to keep it in General changes.

Reply:
I think it should go in Hive, the functions are separated from the main engine by a plugin.